### ISSUE TITLE
[Accessibility]A11y_VSCode_ViewDebug_Keyboard: Keyboard focus goes to checkbox items as a whole which is non-interactive. 

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52400]
**Environment Details:**
VSCode Version : 1.24.0
OS Version : Win10

**Additional Details:**
MAS Violated : MAS 2.1.1

**Repro Steps:**
1. Launch VS Code.
2. Navigate to Activity Bar and select "Debug"(Cntrl+Shift+D) button.
3. Navigate to checkbox  items inside each toolbar using "Tab".

**Actual:**
Keyboard focus moves to overall checkbox  items (div containing all checkbox items) which is non-interactive. Then using downward arrow keys focus goes to items in checkbox .

**Expected:**
Keyboard focus should not move to the items as a whole which is non-interactive. The focus should move to first item in the checkbox.

**Recommendations:**
Remove tab-index from the outer div tag that contains all the checkbox items.
or,
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
The keyboard only users will move to non-interactive elements on the screen which will be time consuming to reach only interactive elements on the page.

**MAS Reference:**
MAS 2.1.1 - https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}

**Attachment For Reference:**
[A11y_VSCode_ViewDebug_Keyboard_KFOnTreeViewOnWhole.pptx](https://github.com/Microsoft/vscode/files/2118138/A11y_VSCode_ViewDebug_Keyboard_KFOnTreeViewOnWhole.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by isidorn** [comment]
Dup of https://github.com/Microsoft/vscode/issues/52000
